rasa/core/channels/console.py

```diff
@@ -111,7 +113,7 @@ def get_user_input(button_question: questionary.Question) -> Optional[Text]:
 async def record_messages(
     server_url=DEFAULT_SERVER_URL,
     auth_token="",
-    sender_id=UserMessage.DEFAULT_SENDER_ID,
+    sender_id=CONSOLE_SESSION_USER_ID,
```

Comment:
this should change at every call of the function, so the `uuid.uuid4().hex` itself should be the default argument (it should not be imported at the module level
```suggestion
    sender_id=uuid.uuid4().hex,
```

Reply:
@ricwo this means every time a new message arrives in the shell we get a new sender_id?
for us it makes more sense to have the same session_id while in the same shell session, so calls to session based components work and maintain context

Reply:
no, since `record_new_messages` is called once per shell session - so this means the sender ID won't change for a given conversation